packages/workspace/src/core/dep-graph/vendor.js

```diff
@@ -2409,10 +2409,10 @@ var tippy = (function(t) {
     return (
```

Comment:
Not sure why this file was changed after `yarn format`.

Reply:
Can we exclude the vendor file from linting?